All_Relationships/utils/datafiles/compound_gene_datafiles.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from tqdm import tqdm_notebook
+
+#Set up the environment
+username = "danich1"
+password = "snorkel"
+dbname = "pubmeddb"
+
+#Path subject to change for different os
+database_str = "postgresql+psycopg2://{}:{}@/{}?host=/var/run/postgresql".format(username, password, dbname)
+
+compound_url = "https://raw.githubusercontent.com/dhimmel/drugbank/7b94454b14a2fa4bb9387cb3b4b9924619cfbd3e/data/drugbank.tsv"
+gene_url = "https://raw.githubusercontent.com/dhimmel/entrez-gene/a7362748a34211e5df6f2d185bb3246279760546/data/genes-human.tsv"
+cbg_url = "https://raw.githubusercontent.com/dhimmel/integrate/93feba1765fbcd76fd79e22f25121f5399629148/compile/CbG-binding.tsv"
+
+
+compound_df = pd.read_table(compound_url)
+
+gene_entrez_df = (
+    pd.read_table(gene_url, dtype={'GeneID': str}) [["GeneID", "Symbol"]]
+    .rename(columns={'GeneID': 'entrez_gene_id', 'Symbol': 'gene_symbol'})
+)
+
+gene_entrez_df['dummy_key'] = 0
+compound_df['dummy_key'] = 0
+pair_df = (
+	gene_entrez_df
+	.merge(compound_df[["drugbank_id", "name", "dummy_key"]], on='dummy_key')
+	.drop('dummy_key', axis=1)
+)
+pair_df.to_csv("data/compound_gene/compound_binds_gene/compound_gene_pairs_binds_full_map.csv", index=False, float_format='%.5g')
```

Comment:
Minor cleanup - I would create a filename variable that you can easily change in the beginning of the code.  Not sure if this file will always be stored/named the same because in that case perhaps you don't want the user to accidentally have access and then change it